packages/bitcore-node/src/config.ts

```diff
@@ -1,38 +1,47 @@
+import fs from 'fs';
 import { cpus, homedir } from 'os';
+import logger from './logger';
 import { ConfigType } from './types/Config';
 import { merge } from './utils';
 import parseArgv from './utils/parseArgv';
 
 const program = parseArgv([], ['config']);
 
 function findConfig(): ConfigType | undefined {
-  let foundConfig;
-  const envConfigPath = process.env.BITCORE_CONFIG_PATH;
-  const argConfigPath = program.config;
-  const configFileName = 'bitcore.config.json';
-  const bitcoreConfigPaths = [
-    `${homedir()}/${configFileName}`,
-    `../../../../${configFileName}`,
-    `../../${configFileName}`
-  ];
-  const overrideConfig = argConfigPath || envConfigPath;
-  if (overrideConfig) {
-    bitcoreConfigPaths.unshift(overrideConfig);
+  let bitcoreConfigPath = program.config || process.env.BITCORE_CONFIG_PATH || '../../bitcore.config.json';
+  if (bitcoreConfigPath[0] === '~') {
+    bitcoreConfigPath = bitcoreConfigPath.replace('~', homedir());
   }
-  // No config specified. Search home, bitcore and cur directory
-  for (const path of bitcoreConfigPaths) {
-    if (!foundConfig) {
-      try {
-        const expanded = path[0] === '~' ? path.replace('~', homedir()) : path;
-        // eslint-disable-next-line @typescript-eslint/no-require-imports
-        const bitcoreConfig = require(expanded) as { bitcoreNode: ConfigType };
-        foundConfig = bitcoreConfig.bitcoreNode;
-      } catch {
-        foundConfig = undefined;
-      }
+  
+  if (!fs.existsSync(bitcoreConfigPath)) {
+    throw new Error(`No bitcore config exists at ${bitcoreConfigPath}`);
+  }
+  
+  const bitcoreConfigStat = fs.statSync(bitcoreConfigPath);
+  
+  if (bitcoreConfigStat.isDirectory()) {
+    if (!fs.existsSync(bitcoreConfigPath + '/bitcore.config.json')) {
+      throw new Error(`No bitcore config exists in directory ${bitcoreConfigPath}`);
     }
+    bitcoreConfigPath += '/bitcore.config.json';
```

Comment:
Better to use `path.join` here. If there's a trailing `/` in the path (e.g. `BITCORE_CONFIG_PATH=/path/to/config/`), then this will be `/path/to/config//bitcore.config.json`...which works but is a little messy and may not work for all OSs. It'd be better to do:
```
bitcoreConfigPath = path.join(bitcoreConfigPath, 'bitcore.config.json);
```
Applies to line 23 as well ( `if (fs.existsSync(path.join(...` ).